script.js

```diff
@@ -3,20 +3,27 @@ const text = document.querySelector(".text");
 document.addEventListener("keydown", (e) => {
```

Comment:
@gaurav-celestial  Some buttons are executing browser default functions.
Eg: F1 is opening up menu, Tab is focusing out of document- window.
Can we handle it here?

Reply:
Done